build-system/tasks/visual-diff/index.js

```diff
@@ -854,6 +856,7 @@ visualDiff.flags = {
   'webserver_debug': '  Prints debug info from the local gulp webserver',
   'percy_agent_debug': '  Prints debug info from the @percy/agent instance',
   'debug': '  Sets all debugging flags',
+  'esm': '  Serve ESM JS',
```

Comment:
Since this task is `visual-diff` and not `serve`, maybe reword this flag description to `Run tests against the module build`?